extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/graal/QuarkusPathLocationScanner.java

```diff
@@ -21,32 +15,46 @@
 
 public final class QuarkusPathLocationScanner implements ResourceAndClassScanner {
     private static final Log LOG = LogFactory.getLog(QuarkusPathLocationScanner.class);
-    /**
-     * File with the migrations list. It is generated dynamically in the Flyway Quarkus Processor
-     */
-    public final static String MIGRATIONS_LIST_FILE = "META-INF/flyway-migrations.txt";
+    private static final String LOCATION_SEPARATOR = "/";
+    private static List<String> applicationMigrationFiles;
+
+    private final Collection<LoadableResource> scannedResources;
+
+    public QuarkusPathLocationScanner(Collection<Location> locations) {
+        this.scannedResources = new ArrayList<>();
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+
+        for (String migrationFile : applicationMigrationFiles) {
+            if (canHandleMigrationFile(locations, migrationFile)) {
+                LOG.debug("Loading " + migrationFile);
```

Comment:
It was preexisiting bug we would need to use `debugf` here. I fixed a few in the Liquibase extension.

Reply:
Done